hail/python/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -ex
+
+cd $(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
+python3=${HAIL_PYTHON3:-python3}
+
+published=$(
+    pip --no-cache-dir search hail \
+     | grep '^hail ' \
+     | sed 's/hail (//' \
+     | sed 's/).*//')
+current=$(
+    grep 'hail_pip_version = ' hail/_generated_version_info.py \
+     | sed -E "s/.*hail_pip_version = \"(.*)\"/\1/")
+
+if [[ "${published}" != "${current}" ]]
+then
+    echo deploying ${current}, was ${published}
+    cp ../build/libs/hail-all-spark.jar hail/
```

Comment:
can we clean up this jar below? Or possibly copy the whole python directory to build/tmp first?

seems wrong to leave big artifacts hanging around with no cleanup rules.

Reply:
I'm really not a fan of copying all the python stuff around. I want to eventually understand the docs build enough to get rid of it. I'll add a trap to cleanup the jar.